config/.env.sample

```diff
@@ -53,6 +53,8 @@ CANVAS_ADMIN_API_TOKEN=
 # CANVAS_NEW_USER_ACCOUNT_ID=1
 # The Canvas custom roles IDs to use when enrolling users to sections (optional)
 # CANVAS_CUSTOM_ROLES={ "Assistant": 34, "Librarian": 21 }
+# The number of matching courses allowed when searching for course sections
+# CANVAS_MAX_SEARCH_COURSES=1000
```

Comment:
This could be changed to 400, reflecting the default in `config.ts`

Reply:
Wups, I forgot to do that.

Reply:
Done. See https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/379/commits/d59bd950214b3827e7e79a68a2e707e014561e99